source/agora/consensus/Fee.d

```diff
@@ -786,7 +786,7 @@ unittest
     auto fee_man = new FeeManager();
 
     Transaction freeze_tx = Transaction(
-        [ Output(Amount(2_000_000L * 10_000_000L), WK.Keys.NODE2.address, OutputType.Freeze) ]);
+        [ Output(Amount(2_000_000.coins), WK.Keys.NODE2.address, OutputType.Freeze) ]);
```

Comment:
just `2_000_000.coins` would work fine, they are of type `Amount` anyways

Reply:
Well spotted.